test/libyul/yulOptimizerTests/stackLimitEvader/function_arg.yul

```diff
@@ -0,0 +1,88 @@
+{
+    {
+        mstore(0x40, memoryguard(128))
+        sstore(0, f(0))
+    }
+    function f(a1) -> v {
+	let a2 := calldataload(mul(2,4))
+	let a3 := calldataload(mul(3,4))
+	let a4 := calldataload(mul(4,4))
+	let a5 := calldataload(mul(5,4))
+	let a6 := calldataload(mul(6,4))
+	let a7 := calldataload(mul(7,4))
+	let a8 := calldataload(mul(8,4))
+	let a9 := calldataload(mul(9,4))
+	let a10 := calldataload(mul(10,4))
+	let a11 := calldataload(mul(11,4))
+	let a12 := calldataload(mul(12,4))
+	let a13 := calldataload(mul(13,4))
+	let a14 := calldataload(mul(14,4))
+	let a15 := calldataload(mul(15,4))
+	let a16 := calldataload(mul(16,4))
+	let a17 := calldataload(mul(17,4))
+	sstore(0, a1)
+	sstore(mul(17,4), a17)
+	sstore(mul(16,4), a16)
+	sstore(mul(15,4), a15)
+	sstore(mul(14,4), a14)
+	sstore(mul(13,4), a13)
+	sstore(mul(12,4), a12)
+	sstore(mul(11,4), a11)
+	sstore(mul(10,4), a10)
+	sstore(mul(9,4), a9)
+	sstore(mul(8,4), a8)
+	sstore(mul(7,4), a7)
+	sstore(mul(6,4), a6)
+	sstore(mul(5,4), a5)
+	sstore(mul(4,4), a4)
+	sstore(mul(3,4), a3)
+	sstore(mul(2,4), a2)
+	sstore(mul(1,4), a1)
+    }
+}
+// ----
+// step: stackLimitEvader
+//
+// {
+//     {
+//         mstore(0x40, memoryguard(0xa0))
+//         sstore(0, f(0))
+//     }
+//     function f(a1) -> v
+//     {
+//         let a2 := calldataload(mul(2, 4))
+//         let a3 := calldataload(mul(3, 4))
+//         let a4 := calldataload(mul(4, 4))
+//         let a5 := calldataload(mul(5, 4))
+//         let a6 := calldataload(mul(6, 4))
+//         let a7 := calldataload(mul(7, 4))
+//         let a8 := calldataload(mul(8, 4))
+//         let a9 := calldataload(mul(9, 4))
+//         let a10 := calldataload(mul(10, 4))
+//         let a11 := calldataload(mul(11, 4))
+//         let a12 := calldataload(mul(12, 4))
+//         let a13 := calldataload(mul(13, 4))
+//         let a14 := calldataload(mul(14, 4))
+//         let a15 := calldataload(mul(15, 4))
+//         let a16 := calldataload(mul(16, 4))
+//         let a17 := calldataload(mul(17, 4))
+//         sstore(0, a1)
```

Comment:
Note that this will produce a stack height error, but we cannot yet move function arguments to memory.

It's important that this isn't changed to ``sstore(0, mload(0x80))`` here (``0x80`` is already reserved as slot for ``a1``, but since we can't properly move, we need to keep the stack error here, i.e. do nothing for this function, which is what is happening)